x/twap/logic.go

```diff
@@ -190,6 +201,12 @@ func recordWithUpdatedAccumulators(record types.TwapRecord, newTime time.Time) t
 	p1NewAccum := types.SpotPriceMulDuration(record.P1LastSpotPrice, timeDelta)
 	newRecord.P1ArithmeticTwapAccumulator = newRecord.P1ArithmeticTwapAccumulator.Add(p1NewAccum)
 
+	// logP0SpotPrice = log_{1.0001}{P_0}
```

Comment:
nit: shouldn't the comment be log_2{P_0}

Reply:
addressed